### PR TITLE
Fix sync repodb signatures

### PR DESCRIPTION
**Pull request by laurentiuNiculae**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/extensions/sync/signatures.go [hunk 1]
```diff
@@ -372,34 +372,34 @@ func (sig *signaturesCopier) syncOCIRefs(localRepo, remoteRepo, digestStr string
...
 				err = sig.repoDB.AddManifestSignature(localRepo, signedManifestDig, repodb.SignatureMetadata{
-					SignatureType:   repodb.NotationType,
-					SignatureDigest: digestStr,
+					SignatureType:   sigType,
+					SignatureDigest: refDigest.String(),
 				})
 			} else {
-				err = repodb.SetImageMetaFromInput(localRepo, digestStr, ref.MediaType,
-					digest, OCIRefBody, sig.storeController.GetImageStore(localRepo),
+				err = repodb.SetImageMetaFromInput(localRepo, subjectStr, ref.MediaType,
```
Comment (peusebiu) [inline pkg/extensions/sync/signatures.go:388]: instead of subjectStr, shouldn't be refDigest ?

Comment (peusebiu) [reply pkg/extensions/sync/signatures.go:388]: the digest of the oci reference itself, not parent.

Comment (laurentiuNiculae) [reply pkg/extensions/sync/signatures.go]: Yes this is correct. I've updated it

**Review by andaaron — APPROVED** [review]

**Review by peusebiu — APPROVED** [review]
lgtm

**Comment by andaaron** [comment]
Need this to complete https://github.com/project-zot/zot/issues/1434

**Pull request merged**